.gitignore

```diff
@@ -9,3 +9,6 @@ NugetPackTest/
 examples/bin/
 examples/obj/
 .vs
+src/test/FunctionTest/FunctionTest.sln
```

Comment:
ignore a src dir is not a good idea.

Reply:
Since I'm not sure about this .sln file. This was generated by visual studio while testing. I will add this .sln file for my further debug.